site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.0 also introduces an improved status tailing experience which surfaces relevant information while waiting for resources to reconcile.
+
+# The `app` commands
+These commands aim to help users manage App CRs on the cluster better. Lets dig in and see what they look like!
+
```

Comment:
Should we add the command name as heading before start talking about it. This will be helpful if reader wants to glance. e.g. `### app list `

Reply:
We could, but I decided against it so that it does not look like documentation
I would be open to the idea though. 

Also we don't really talk enough about `package kick/pause/status` so might be weird to not have headings for a few commands

Reply:
Cool